backbone.layoutmanager.js

```diff
@@ -262,11 +262,6 @@ var LayoutManager = Backbone.View.extend({
     return this.__manager__.renderDeferred.promise();
   },
 
-  // Proxy `then` for easier invocation.
-  then: function() {
-    return this.promise().then.apply(this, arguments);
```

Comment:
I'm still looking into the issue.  It seems to be because of the property named `then`, but we're only passing the `promise()` return value into Bluebird resolve, so why should that matter.